.github/workflows/lgtm-approval.yml

```diff
@@ -0,0 +1,22 @@
+name: Approve PR on /lgtm comment
+
+on:
+  workflow_dispatch: #manual testing
+  issue_comment:
+    types: [created]
```

Comment:
please add condition to trigger only if "lgtm in comment"
e/g
```
    if: |
      contains(github.event.comment.body, '/lgtm')
```

also should be triggered on `edited`

Reply:
The actions already checks if /lgtm is int he comment: https://github.com/opendatahub-io/opendatahub-tests/pull/153/files#diff-8adeafbcbe79cc05a3ae753fd95a88451697a3356e53b468f8a7f565c5c737d1R13
Why would we edit a comment to add a lgtm? that seems unlikely to be honest

Reply:
1. i think adding the check on the content of the comment at the top makes it more clear
2. a user can edit a comment and add "lgtm"

Reply:
I've added the edit event, I still think it is not a likely scenario though :) 
The `if` conditional is not supported on the definition of the event as far as I can see.